tor/helpers/flair.py

```diff
@@ -61,14 +83,31 @@ def update_user_flair(post, config):
         user_flair = flair_text
 
     if 'Γ' in user_flair:
-        # take their current flair and add one to it
-        new_flair_count = int(user_flair[:user_flair.index('Γ') - 1])
+        # extract their current flair and add one to it
+        new_flair_count = int(user_flair[:user_flair.index('Γ') - 1]) + 1
```

Comment:
Is it possible for someone/thing to have a malformed flair, that this cast could then fail on?

Reply:
If the flair has the gamma inside it, it's a reasonable assumption that it has been set correctly. We've only had one instance of flair that was "malformed", in which case Reddit for some reason didn't accept the call to actually set it in the first place. If that happens, it gets intercepted in `process_done` and we don't even get this far. Personally I don't think it's an issue -- this particular bit has been running since the very first release without a problem.

Reply:
In that case it's probably fine

Reply:
¯\_(ツ)_/¯